src/Brahma.FSharp.OpenCL.Core/CommandQueueProvider.fs

```diff
@@ -80,15 +76,13 @@ type ComputeProvider(context: Context, device: Device) as this =
 
         processor
 
-    member val CommandQueue = getNewProcessor () with get
-
-    member private this.HandleFree(free: IFreeCrate) =
+    static member private HandleFree(free: IFreeCrate) =
```

Comment:
Why ```static```? How can I handle messages without an instance of handler?

Reply:
Because `CommandQueueProvider` is a factory that produces MBP that uses these handlers within itself.